code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -66,33 +66,30 @@
 	projectiletype = /obj/item/projectile/beam/disabler
 	ranged_cooldown_time = 20
 	projectilesound = 'sound/weapons/taser2.ogg'
+	loot = list(/obj/effect/decal/cleanable/robot_debris, /obj/item/weapon/ore/bluespace_crystal/artificial)
+	del_on_death = 1
 	var/resources = 0 //Resource points, generated by consuming metal/glass
 
 /mob/living/simple_animal/hostile/swarmer/Login()
 	..()
 	src << "<b>You are a swarmer, a weapon of a long dead civilization. Until further orders from your original masters are received, you must continue to consume and replicate.</b>"
-	src << "<b>Ctrl + Click provides most of your swarmer specific interactions, such as cannibalizing metal or glass, destroying the environment, or teleporting mobs away from you."
+	src << "<b>Clicking on any object will try to consume it, either deconstructing it into its components, destroying it, or integrating any materials it has into you if successful.</b>"
+	src << "<b>Ctrl-Clicking on a mob will attempt to remove it from the area and place it in a safe environment for storage.</b>"
 	src << "<b>Objectives:</b>"
 	src << "1. Consume resources and replicate until there are no more resources left."
-	src << "2. Ensure that the station is fit for invasion at a later date, do not perform actions that would render it dangerous or inhospitable."
-	src << "3. Biological resources will be harvested at a later date, do not harm them."
+	src << "2. Ensure that this location is fit for invasion at a later date; do not perform actions that would render it dangerous or inhospitable."
+	src << "3. Biological resources will be harvested at a later date; do not harm them."
 
 /mob/living/simple_animal/hostile/swarmer/New()
 	..()
 	verbs -= /mob/living/verb/pulled
+	deathmessage = "The [src] explodes with a sharp pop!"
```

Comment:
it'll come out as swarmer (543); also this already precludes admins editing the name because this is set in new and not when they're about to die.